Assets/Scripts/PauseMenu.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
System.Collections and System.Collections.Generic not needed

Reply:
Ok. unity adds that automatically and I didn't bother removing it.